gdal/ogr/ogrsf_frmts/amigocloud/ogramigoclouddatasource.cpp

```diff
@@ -48,7 +48,9 @@ OGRAmigoCloudDataSource::OGRAmigoCloudDataSource() :
     bUseHTTPS(false),
     bMustCleanPersistent(false),
     bHasOGRMetadataFunction(-1)
-{}
+{
+    CPLSetConfigOption( "GDAL_HTTP_USERAGENT", "gdal/AmigoCloud build:2.x" );
```

Comment:
Wouldn't it be better to use the output of GDALVersionInfo("RELEASE_NAME") instead of hardcoding 2.x ?

Reply:
Yes, it would. I couldn't find the way to get that info. I'll make a change. Thanks.

Reply:
And actully instead of messing with the global option GDAL_HTTP_USERAGENT, it would be better to pass "USERAGENT=xxx" as an option of CPLHTTPFetch()

Reply:
I will do that.